packages/protocol/src/action-protocol/element-hover.ts

```diff
@@ -13,47 +13,79 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
-import { isObject, isString } from '../utils/typeguard-util';
-import { generateRequestId, isActionKind, RequestAction, ResponseAction } from './base-protocol';
+import { Bounds } from 'sprotty-protocol';
+import * as sprotty from 'sprotty-protocol/lib/actions';
+import { hasObjectProp, hasStringProp } from '../utils/type-util';
+import { Action, RequestAction, ResponseAction } from './base-protocol';
 import { SModelRootSchema } from './model-structure';
-import { Bounds } from './types';
 
 /**
  * Triggered when the user hovers the mouse pointer over an element to get a popup with details on that element.
  * This action is sent from the client to the server. The response is a `SetPopupModelAction`.
+ * The corresponding namespace declares the action kind as constant and offers helper functions for type guard checks
+ * and creating new `RequestPopupModelActions`.
  */
-export class RequestPopupModelAction implements RequestAction<SetPopupModelAction> {
-    static readonly KIND = 'requestPopupModel';
-    readonly kind = RequestPopupModelAction.KIND;
+export interface RequestPopupModelAction extends RequestAction<SetPopupModelAction>, sprotty.RequestPopupModelAction {
+    /**
+     * The unique action kind.
+     */
```

Comment:
I'd spare such comments, also if it hurts to be "incomplete". But imho they just add noise and add no information.